yaccparser.py

```diff
@@ -711,6 +713,47 @@ def p_multiplyinstruction(p):
     # so we just pack it in a bytes object
     p[0] = (struct.pack("<I", p[0]), None)
 
+def p_multiplylonginstruction(p):
+    """multiplylonginstruction : OPMULL logmnemonic flagscondandspace REG COMMA REG COMMA REG COMMA REG
+                               | OPMULL logmnemonic flagscondandspace REG COMMA REG COMMA SHARP CONST"""
+    global currentMnemonic
+    if len(p) == 10:
+        raise YaccError("Une instruction {} ne peut recevoir de constante comme dernier argument, seulement un registre.".format(currentMnemonic))
+
+    p[0] = 9 << 4
+    p[0] |= 1 << 23
+    assert len(p) == 11         # Check if we have 4 registers
```

Comment:
Since the case len(p)==10 was tested before, it is technically impossible for this to occurs.